framework/src/node/synchronizer/synchronizer.ts

```diff
@@ -91,54 +94,59 @@ export class Synchronizer {
 	}
 
 	public async run(receivedBlock: Block, peerId: string): Promise<void> {
-		if (this.isActive) {
-			throw new Error('Synchronizer is already running');
+		if (this._mutex.isLocked()) {
+			this.logger.debug('Synchronizer is already running.');
```

Comment:
```suggestion
			this.logger.info('Synchronizer is already running.');
```

Reply:
this will happen a lot, so i kept it as `debug`

Reply:
Sync is triggered only when chain is out of sync.. so why it happens a lot? i can understand it can happen during initial time before certain height, but after that it shouldn't trigger sync if blocks are propagating? 
Felt the log will be useful, its ok either way.